### PR TITLE
Add progress indicator

### PR DESCRIPTION
**Pull request by richtier**
[this task](https://uktrade.atlassian.net/browse/TT-1055)
progress

### CODE REVIEW

File: core/sass/enrolment.scss [hunk 1]
```diff
@@ -119,3 +121,30 @@ body.js-enabled {
...
+#content {
+  nav.breadcrumbs {
+    padding-bottom: 0;
+    padding-left: 15px;
+  }
+}
+
+#progress-column {
+  float: right;
+  border-top: 5px solid #BF0000;
```
Comment (eadpearce) [inline core/sass/enrolment.scss]: could this not be `$flag-red` or does it have to be a specific new colour that was in the designs that we don't already use?

File: core/sass/enrolment.scss [hunk 2]
```diff
@@ -119,3 +121,30 @@ body.js-enabled {
...
+    margin: 0;
+  }
+}
+
+#start-now-button {
+  display: block;
+  margin-top: 15px;
+  width: 145px;
+  &:active {
+    background-color: #004FB3;
```
Comment (eadpearce) [inline core/sass/enrolment.scss]: same here. do we not already have a colour variable we can use

File: enrolment/templates/enrolment/progress.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<div class="steps-progress-indicator" aria-label="Process steps">
+  <progress class="visually-hidden" max="{{ wizard.steps.count }}" value="{{ step_number }}">You are on step {{ steps.step1 }} of {{ steps.count }} in this process</progress>
```
Comment (eadpearce) [inline enrolment/templates/enrolment/progress.html]: omg i didn't know this element existed. super neat

File: enrolment/templates/enrolment/start.html [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    <section class="container">
+        {% include 'enrolment/breadcrumbs.html' %}
+        <div class="column-one-half">
+            <h1 class="heading-xlarge">Create an account</h1>
+            <p>We'll email you a confirmation code while creating your account. Make sure you have access to your email.</p>
+            <p><a href="{{ sso_login_url }}">Sign in</a> if you have an account.</p>
+            
+            <div class="steps-progress-indicator" id="start-page-progress-inidicator" aria-label="Process steps">
+              <p id="steps-progress-explaination" class="visually-hidden">List of steps required to complete the process:</p>
+              <ol aria-labelledby="steps-progress-explaination">
```
Comment (eadpearce) [inline enrolment/templates/enrolment/start.html]: "explanation"

File: enrolment/templates/enrolment/start.html [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+                <li class="current">
+                    <span class="visually-hidden">Current:</span>
+                    <span>
+                        Select your business type
+                        <a id="start-now-button" class="button" href="{% url 'enrolment' step='company-type' %}">Start now</a>
+                    </span>
+                    
+                </li>
+                <li><span>Enter your email and set a password</span></li>
```
Comment (eadpearce) [inline enrolment/templates/enrolment/start.html]: are all these `span`s necessary?

File: core/sass/partials/_progress.scss [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+      border-left-color: $white;
+
+      span {
+        border: none;
+      }
+    }
+  }
+
+  // Adds the separating lines.
+  span {
```
Comment (eadpearce) [inline core/sass/partials/_progress.scss]: ah i see. the `span`s are used for styling. you'd be better off using a class and adding it to the parent `li`s instead

File: enrolment/templates/enrolment/start.html (hunk 4 above)
Comment (richtier) [reply enrolment/templates/enrolment/start.html]: image

File: enrolment/templates/enrolment/start.html (hunk 5 above)
Comment (richtier) [reply enrolment/templates/enrolment/start.html]: si, the numbers go away without them

image

Comment (richtier) [reply enrolment/templates/enrolment/start.html]: image

**Pull request merged**